### PR TITLE
#158689317 setup database

### PR DESCRIPTION
**Pull request by tomiadebanjo**
### What does this PR do?
- It gives the developer a database to connect to be used to get and post data through the endpoints

#### Description of Task to be completed?
- Create tables on pgadmin
- Install Pg client on node application
- Successfully Tested connection between the application and the database

#### What are the relevant pivotal tracker stories?
[#158689317](https://www.pivotaltracker.com/story/show/158689317)

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -14,6 +17,13 @@ app.use('/', router);
 
 app.get('/', (req, res) => {
   res.status(200).json('Welcome to Ride My Way!');
+  // database connection test
+  // pool.query('SELECT * FROM users', (err, result) => {
+  //   if (err) {
+  //     return console.log('error running query', err);
+  //   }
+  //   res.json({ users: result.rows });
+  // });
```
Comment (Ruqoyah) [inline server/app.js]: Take out commented code

File: server/models/indexdb.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import pg from 'pg';
+import dotenv from 'dotenv';
+
+dotenv.config();
+const connectionString = 'postgres://postgres:temitope@localhost:5432/ridemyway';
```
Comment (Ruqoyah) [inline server/models/indexdb.js]: If I'm not mistaken, you pushed your database password online right?

**Comment by Ruqoyah** [comment]
Kindly improve on **What does this PR do?** as I don't really understand what it means. Also, I actually did not see where you created tables in your file changes.

**Review by Ruqoyah — CHANGES_REQUESTED** [review]

File: server/app.js (hunk 1 above)
Comment (tomiadebanjo) [reply server/app.js]: okay i will

File: server/models/indexdb.js (hunk 2 above)
Comment (tomiadebanjo) [reply server/models/indexdb.js]: yes i did .. I will fix that

File: server/app.js [hunk 3]
```diff
@@ -2,6 +2,9 @@ import express from 'express';
 import bodyParser from 'body-parser';
 import logger from 'morgan';
 
+
+import pool from './models/indexdb';
```
Comment (Ruqoyah) [inline server/app.js]: Why did you have this here?

**Review by Ruqoyah — APPROVED** [review]

**Pull request merged**